tools/tidb-binlog-cluster.md

```diff
@@ -615,7 +617,7 @@ Usage of binlogctl:
 
 ## 版本升级方法
 
-Cluster 版本的 TiDB-Binlog 不兼容 Kafka 版本以及 Local 版本，TiDB 如果升级到新版（v2.0.8-binlog、v2.1.0-rc.5 及以上版本）后只能使用 Cluster 版本的 TiDB-Binlog；如果在升级前已经使用了 Kafka／Local 版本的 TiDB-Binlog，必须将其升级到 Cluster 版本。
+新版本的 TiDB（v2.0.8-binlog、v2.1.0-rc.5 及以上版本）不兼容 TiDB-Binlog 的 Kafka 版本以及 Local 版本，TiDB 如果升级到新版后只能使用 Cluster 版本的 TiDB-Binlog；如果在升级前已经使用了 Kafka／Local 版本的 TiDB-Binlog，必须将其升级到 Cluster 版本。
```

Comment:
It would be better and clearer to tell users the corresponding relationship between the TiDB version and the TiDB-Binlog version.